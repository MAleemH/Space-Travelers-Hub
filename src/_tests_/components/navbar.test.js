import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../../components/navbar/navbar';
import store from '../../redux/store';

describe('Missions', () => {
  it('Renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter store={store}>
          <Navbar />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
