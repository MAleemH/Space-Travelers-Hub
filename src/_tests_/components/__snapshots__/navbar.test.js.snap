// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions Renders correctly 1`] = `
<nav>
  <a
    className="nav-title"
    href="/"
  >
    <img
      alt="Logo-Img"
      height="50"
      src="logo.png"
      width="50"
    />
    <h1
      className="web-title"
    >
      Space Travelers' Hub
    </h1>
  </a>
  <ul
    className="nav-list"
  >
    <li
      className="nav-items"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Rockets
      </a>
    </li>
    <li
      className="nav-items"
    >
      <a
        className=""
        href="/missions"
        onClick={[Function]}
      >
        Missions
      </a>
    </li>
    <li
      className="nav-items"
    >
      <a
        className=""
        href="/profile"
        onClick={[Function]}
      >
        Profile
      </a>
    </li>
  </ul>
</nav>
`;
